### PR TITLE
Update proposed CoCC charter

### PR DESCRIPTION
**Pull request by joannalee333**
Proposing a draft CNCF Code of Conduct Committee Charter. Note that the names of current committee members has been removed because current members can be listed in a separate file / webpage

### CODE REVIEW

File: coc-committee-description-proposed.md
Comment (carolynvs) [reply coc-committee-description-proposed.md:22]: One suggestion would be that "After 3 events where a representative is unable to participate (excluding absences due to a conflict of interest), they are encouraged to step down so that another representative with more time to dedicate to the committee can be elected"

File: coc-committee-description-proposed.md
Comment (joannalee333) [reply coc-committee-description-proposed.md:41]: Yes, the Marketing Committee is sometimes asked to deal with conflicts that can overlap with CoC incidents - e.g., complaints from one vendor about another vendor rep's behavior or comments

File: coc-committee-description-proposed.md
Comment (joannalee333) [reply coc-committee-description-proposed.md:22]: Good suggestion, @carolynvs. I'll discuss with the working group co-chairs today

File: coc-committee-charter-proposed.md
Comment (nikhita) [inline coc-committee-charter-proposed.md:32]: Is TAG Security here a typo or is that expected?

File: coc-committee-charter-proposed.md
Comment (nikhita) [inline coc-committee-charter-proposed.md:34]: > remaining CoC Committee members may vote to remove them

How is this voting determined? For example, does this require a super majority vote among the remaining CoCC members?

Comment (nikhita) [reply coc-committee-charter-proposed.md:34]: If it's helpful, here is what the Kubernetes Steering Committee follows for a no confidence removal - https://github.com/kubernetes/steering/blob/main/charter.md#no-confidence

File: coc-committee-charter-proposed.md
Comment (joannalee333) [reply coc-committee-charter-proposed.md:32]: Yes, a typo. I'll fix that. Should be TAG-Contributor Strategy

File: coc-committee-charter-proposed.md
Comment (joannalee333) [reply coc-committee-charter-proposed.md:34]: Decisions will be made by consensus or majority vote

File: coc-committee-charter-proposed.md
Comment (dims) [inline coc-committee-charter-proposed.md:60]: a couple of nits @joannalee333 !

- `shall not participate in the investigation` -> `shall not participate in any investigations`
- `regading` -> `regarding`

Comment (joannalee333) [reply coc-committee-charter-proposed.md:67]: Thanks, @dims. I've fixed those

Comment (dims) [reply coc-committee-charter-proposed.md:67]: 🙏

**Comment by tpepper** [comment]
I generally like the latest update.  One thing to keep in mind is that aside from the "active participant" requirement, the additional eligibility criteria are subjective, and this sometimes causes objections by people who want simple, logical, concrete eligibility metrics.  In practice though it is not an issue.  In the Kubernetes project case since it is the constrained voter set of the Steering Committee, it is implicit that a call for non-eligibility of a nominee can happen with that body and happen ahead of it voting, or simply be a part of their vote, ie: not voting for people they don't feel show their desired level of the responsible/thoughtful/caring attributes.  This ends up the same as any other metric a voter might evaluate (eg: perceived to have the time, experience, conflicts of interest, etc., etc., etc.).   The same vetting should work here via the TOC.

**Comment by joannalee333** [comment]
The Co-Chairs met on May 4 and approved this PR, subject to one edit to clarify that the CoC Committee can share information with the TOC in a similar manner to how information is shared with the GB. However we didn't have quorum (Arun, Emily, and Matt were present but no elected Maintainer representative was present).

@cblecker and @anvega, do you also approve? If so, could one of you please merge this PR? Thanks!

**Review by anvega — APPROVED** [review]

**Pull request merged**
